runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs

```diff
@@ -290,3 +281,96 @@ pub fn sum_n() {
         value_return(data.len() as u64, data.as_ptr() as u64);
     }
 }
+
+fn build_string(n: u64) -> String {
```

Comment:
Is it just `String::from_utf8(vec![b'a'; n]);`?
@mikhailOK

Reply:
Actually you can do `"a".repeat(n)`

Reply:
Fixed.